src/sentry/eventstore/filter.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+
+class Filter(object):
+    conditions = []
+    filter_keys = {}
+    start = None
+    end = None
+
+    def __init__(self, start=None, end=None, conditions=None, filter_keys=None):
+        self.start = start
+        self.end = end
+
+        if conditions:
```

Comment:
Do we need to split between filter and conditions for the end user? Is there a way to avoid this ?